src/environment.ts

```diff
@@ -83,15 +81,9 @@ export class SecureEnvironment {
         if (isUndefined(options)) {
             throw new Error(`Missing SecureEnvironmentOptions options bag.`);
         }
-        const { rawGlobalThis, secureGlobalThis, distortionCallback } = options;
+        const { distortionCallback, secureGlobalThis } = options;
         this.distortionCallback = distortionCallback;
         this.secureGlobalThis = secureGlobalThis;
-        // These are foundational things that should never be wrapped but are equivalent
-        // TODO: revisit this, is this really needed? what happen if Object.prototype is patched in the sec env?
```

Comment:
well, the problem here is really about object identity. Specifically, if the sandbox creates an obj like `const foo = {}`, and calls a secure proxy function by passing `foo` as argument, the outer realm should be able to test that `foo instanceof Object` is `true`. Without the line above, it will not work. The same for the inverse... e.g.: `document.body.dataset instanceof Object` should be `true` from within the sandbox.

My note was really about other things that should be mapped, not only there 4.

Reply:
Ah. Okay.